libcontainer/cgroups/systemd/apply_systemd.go

```diff
@@ -159,8 +160,36 @@ func UseSystemd() bool {
 			}
 		}
 
+		// Assume we have the ability to start a transient unit as a slice
+		// This was broken until systemd v229, but has been back-ported on RHEL environments >= 219
+		// For details, see: https://bugzilla.redhat.com/show_bug.cgi?id=1370299
```

Comment:
I would strongly suspect that other distros will not work with systemd <= 229, we can verify the other distros when we consume this in kubernetes.  i doubt others have patched their systemd.